.github/workflows/azure-login-pr-check.yml

```diff
@@ -5,6 +5,10 @@ on:
     branches:
       - master
       - 'releases/*'
+
+permissions:
+  contents: read
```

Comment:
I still don't understand why to add this line.

Reply:
Try creating a private repository and import this repository, actions/checkout will fail because it requires this permission.

Reply:
Could you explain how you "import" this repo in your private repo? We do not intend for this workflow to be reused in other repos.

Reply:
1. visit https://github.com/new/import 
2. enter https://github.com/Azure/login/ as the clone url

I mostly do it if i'm going to be looking for vulnerabilities in a repository...

Reply:
@jsoref, thanks for sharing this with us. However, the target of this workflow file is only for this repo.  An "imported" repo is a fresh new one with all commits. It's out of our scope.